introduction.md

```diff
@@ -4,6 +4,11 @@ Johan here. I'm doing the GitHub class on Code School.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?